evadb/udfs/sentence_feature_extractor.py

```diff
@@ -32,6 +32,7 @@ def setup(self):
             "sentence-transformers/all-MiniLM-L6-v2"
```

Comment:
Do we converge on which UDF to keep this or the `sentence_transformer_feature_extractor`? I think they are the same model, but the other implementation looks cleaner than this.

Reply:
Which one did you add?

Reply:
Let's just call this `text_feature_extractor`?

Reply:
Sure. I added this one, but i think other one is much simpler and also the device issue will disappear. I just found that this UDF’s implementation has a bug.